server/views/global/typography/typography.config.js

```diff
@@ -78,7 +78,7 @@ export const context = {
       example: 'Library Item Group Info Label',
       size: '20px',
       spacing: '40',
-      lineHeight: '32px',
+      lineHeight: '30px',
```

Comment:
yeah, that's what I mean – I started looking at this a while back (when I was doing the rest of the json config stuff) but it wasn't straightforward – as I remember, the main pain point was getting a list of font-family names (e.g. `'helvetica neue', helvetica, arial`) as a property for a json key without everything blowing up.

I'll give it another go soon, but don't want to drop a couple of hours on it for the sake of updating a couple of line-heights.